test.js

```diff
@@ -100,6 +100,7 @@ assert.format('%k', null, '18');
 assert.format('%L', '067');
 assert.format('%l', null, ' 6');
 assert.format('%-l', null, '6');
+assert.format("%-y", "1", null, new Date('2001-02-03T04:05:06'))
```

Comment:
There should be a test for years that don't begin with `0` here too, something like this:

```javascript
assert.format("%-y", "23", null, new Date('2023-02-03T04:05:06'))
```

Reply:
Hello, I have added the test case and fixed two issues.